doc/nrf/gs_installing.rst

```diff
@@ -30,21 +33,26 @@ Install the required tools
 **************************
 
 The installation process is different depending on your operating system.
+The installation steps are based on the Install dependencies section in Zephyr's :ref:`zephyr:getting_started`.
```

Comment:
Is this information really relevant? We do not want users to click that link.

Reply:
I gues slines 20-21 cover this information already, so this sentence is not needed.